data/README.md

```diff
@@ -1,2 +1,16 @@
-# Dataset
+# Data Directory
 
+The following directory contains a variety of data used in the SM3-Text-to-Query project. 
+
+Below you will find a brief description of the specific subdirectories.
+
+
+- `./synthea_database` contains the Synthea data used as the basis for PostgreSQL, Graph DB (RDF), Neo4j, and MongoDB.
```

Comment:
The paths are still wrong. Please check all things carefully before submitting it...